plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/IgniteQueryRunner.java

```diff
@@ -80,14 +81,16 @@ public static void main(String[] args)
     {
         Logging.initialize();
 
-        DistributedQueryRunner queryRunner = createIgniteQueryRunner(
-                TestingIgniteServer.getInstance().get(),
-                ImmutableMap.of("http-server.http.port", "8080"),
-                ImmutableMap.of(),
-                TpchTable.getTables());
+        try (SharedResource.Lease<TestingIgniteServer> instance = TestingIgniteServer.getInstance()) {
```

Comment:
We can't connect the Ignite server if we close this like this.

Reply:
Correct me if I am wrong, my understanding is, here is only the test for createIgniteQueryRunner method with the TestingIgniteServer, so before the main process exit we need close the resource.

Reply:
This isn't a test. We use this method when we want to access Trino or Ignite server in local environment.

Reply:
ha, yes, I just notice it's only for allowing access the server in local environment.

Reply:
We should remove the close logic in `PhoenixQueryRunner`